.gitignore

```diff
@@ -23,4 +23,9 @@ erl_crash.dump
 id3-*.tar
 
 # Rustler creates native files inside /priv. We'll ignore that.
-/priv/native/*
\ No newline at end of file
+/priv/native/*
+
+native/id3/.cargo/config
+
+.DS_Store
+priv/.DS_Store
```

Comment:
I think this should go to your local `$GIT_DIR/info/exclude` or `core.excludesFile`, since this is specific to macOS systems.
https://git-scm.com/docs/gitignore